### ISSUE TITLE
compilation error with release v3.0.0

### ISSUE THREAD

**Issue by perscrew** [EddyVerbruggen/nativescript-local-notifications#104]
Hi, I got an error with the latest release 3.0.0.

The global import is not working correctly : 

```
import * as LocalNotifications from "nativescript-local-notifications";
```

The compiler don't recognize the schedule and other methods of your library.

**Comment by vladwelt** [comment]
Hi, try the next:

`
import { LocalNotifications } from "nativescript-local-notifications";
`

**Comment by EddyVerbruggen** [comment]
Thanks for the readme fix in #105!

**Comment by EddyVerbruggen** [comment]
Not intentionally. But you can browse git history to find evidence and put it back if you like.
